STM32Cube/Tasks/trajectory.c

```diff
@@ -73,7 +74,10 @@ void trajectory_task(void* argument) {
                 if (prev_alt != INT_MAX) {
                     float vely_raw = (altTime.alt - prev_alt) * 1000.0 /
                                  (altTime.time - prev_time);
-                    float vely = second_order_lowpass_filter(vely_raw, filter_data, VELOCITY_FILTER_ALPHA);
+
+                    //float vely = second_order_lowpass_filter(vely_raw, filter_data, VELOCITY_FILTER_ALPHA);
+                    float vely = moving_average_filter(vely_raw, filter_data);
```

Comment:
I think this might effectively increase the number of timeout readings, which I assume is intended

Reply:
yes
should reduce false positives during coast